admin_manual/issues/impersonate_users.rst

```diff
@@ -0,0 +1,49 @@
+===================
+Impersonating Users
+===================
+
+Sometimes you may need to use your ownCloud installation as another user, whether to help users debug an issue or to get a better understanding of what they see when they use their ownCloud account.
+The ability to do so is a feature in ownCloud 10.0, via a `Marketplace` app called "**Impersonate**". 
+
+.. note::
+   This functionality is available only to administrators.
+
+Installing the Application
+--------------------------
+
+To install it, from the Market in your ownCloud installation, filter down the list of available apps to the **Tools** category, and in the filtered list click on the **Impersonate** app. 
```

Comment:
Not sure if we need installation instructions as this is exactly the same for **all** apps in the future. You can leave it here but this should actually be documented in a generalized way for the marketplace.